crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -650,6 +661,37 @@ pub async fn resolve(
             )
         }
         Request::Module { module, path } => {
+            // Warn if the package is known not to be supported by Turbopack at the moment.
+            if UNSUPPORTED_PACKAGES.contains(module) {
+                UnsupportedModuleIssue {
```

Comment:
Can we isolate the source position for this import? https://github.com/vercel/turbo/blob/wbinnssmith/warn-unsupported-modules/crates/turbopack-core/src/issue/mod.rs#L117-L122

Reply:
Yeah, I'd really like that. Unfortunately even the context path isn't the importing file — it's the dirname (I imagine because this is the context necessary for resolution). I'm going to see if I can move issue creation closer to the import site, but this was the most centralized place for now.